app/models/team.rb

```diff
@@ -25,16 +25,29 @@ def invite(user_params, inviter)
     transaction do
       user.save!(validate: false) unless user.persisted?
 
-      team_invite = team_users.create!(
-        user: user,
-        role: "user",
-        status: "invited",
-        invitation_code: SecureRandom.urlsafe_base64,
-        invited_at: Time.zone.now,
-        invited_by: inviter.id
-      )
-
-      UserMailer.team_invite(team_invite, inviter).deliver_later
+      # do not invite if the user is already on the team
+      unless team_users.pluck(:user_id).include?(user.id)
```

Comment:
can we use guard clauses here to simplify reading of this rather than nesting i.e. `return unless team_users.pluck(:user_id).include?(user.id)`